### DISCUSSION TITLE
Growth rates and relative abundance

### DISCUSSION THREAD

**Discussion by dinesh-kumar-k-b** [micom-dev/micom#122] [Q&A] [answered]
In a 3-member community ABC, where we want to decrease the growth of A by adding a probiotic species, what should be the relative abundance of the probiotic species to be added? 

As relative abundance is an input parameter in MICOM, is decreased growth of A independent of its relative abundance?

Thanks in advance.

**Comment by cdiener** [comment, marked as answer]
It should be the expected *steady state abundance* of the organism. So the expected relative abundance during exponential growth.
